Extensions/keyboardShortcut.js

```diff
@@ -109,7 +109,7 @@
     }
 
     function clickQueueButton() {
-        document.querySelector(".control-button-wrapper .spoticon-queue-16").click();
+        document.querySelector(".control-button-wrapper > button.Button-sc-1dqy6lx-0").click();
```

Comment:
It seems that the class being used is not mapped, so let's try to use a generic way of accessing this button, ie, simply this:
```js
document.querySelector(".control-button-wrapper > button").click();
```

Reply:
I see. Also, I discovered that this selector would select the "Remove" button first, so i make it a bit more specific.

Reply:
The remove button? Which remove button? If that's the case then maybe we do need to be more specific.

Reply:
The one on the `.main-nowPlayingBar-left` to make the auto-suggested song not to play again
![image](https://user-images.githubusercontent.com/35028055/150987652-e80a0330-da1e-45b3-ae75-c17df671f952.png)